server/i18n/en/characteristics.json

```diff
@@ -0,0 +1,23 @@
+{
+  "isSingle": "Single room",
+  "isFullyFm": "All furnishings and bedding supplied by FM",
+  "hasCrib7Bedding": "Crib 7 rated bedding",
+  "hasSmokeDetector": "Smoke/heat detector",
+  "isTopFloorVulnerable": "On top floor and vulnerable",
+  "isGroundFloorNrOffice": "On ground floor close to office",
+  "hasNearbySprinkler": "Near to sprinkler",
+  "isArsonSuitable": "Suitable for arson risks",
+  "isArsonDesignated": "Designated arson room",
+  "hasArsonInsuranceConditions": "Not insured for arson risk",
+  "isSuitedForSexOffenders": "Suitable for sex offenders",
+  "hasEnSuite": "En suite",
+  "isWheelchairAccessible": "Can be accessed in wheelchair",
+  "hasWideDoor": "Door >= 90cm wide",
```

Comment:
Are we already using this translation? It strikes me as a little technical (compared to e.g. "Door 90cm or wider")